CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 
 ### 🚩 Deprecations 🚩
 
+- `datadogexporter`: Deprecate `instrumentation_library_metadata_as_tags` (#11135)
```

Comment:
ah, sorry, missed this but we would also need a note under "enhancements" about the new option